MySQL 8.0 build fails on macOS with "Could not run sw_vers" message

### Describe the bug

```
❯ nix build nixpkgs#mysql80
error: builder for '/nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv' failed with exit code 1;
       last 10 log lines:
       >
       > CMake Error at cmake/package_name.cmake:98 (MESSAGE):
       >   Could not run sw_vers
       > Call Stack (most recent call first):
       >   cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
       >   CMakeLists.txt:653 (INCLUDE)
       >
       >
       > -- Configuring incomplete, errors occurred!
       > See also "/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build/CMakeFiles/CMakeOutput.log".
       For full logs, run 'nix log /nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv'.
```

`/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build/CMakeFiles/CMakeOutput.log` doesn't exist.

`nix log /nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv`:

<details>

```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/8qlc0wxzndifrnqrckfjq5bznif1fx73-mysql-8.0.26.tar.gz
source root is mysql-8.0.26
setting SOURCE_DATE_EPOCH to timestamp 1625127339 of file mysql-8.0.26/sql/sql_yacc.h
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
applying patch /nix/store/sn3mhjgqrrplgag8grhc91g5a9lcp1z2-abi-check.patch
patching file cmake/do_abi_check.cmake
patching file include/nostdinc/stdint.h
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/doc/mysql -DCMAKE_INSTALL_INFODIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/include -DCMAKE_INSTALL_SBINDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/strip -DCMAKE_RANLIB=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/ranlib -DCMAKE_AR=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26 -DCMAKE_SKIP_BUILD_RPATH=OFF -DFORCE_UNSUPPORTED_COMPILER=1 -DWITH_ROUTER=OFF -DWITH_SYSTEM_LIBS=ON -DWITH_UNIT_TESTS=OFF -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock -DMYSQL_DATADIR=/var/lib/mysql -DINSTALL_INFODIR=share/mysql/docs -DINSTALL_MANDIR=share/man -DINSTALL_PLUGINDIR=lib/mysql/plugin -DINSTALL_INCLUDEDIR=include/mysql -DINSTALL_DOCREADMEDIR=share/mysql -DINSTALL_SUPPORTFILESDIR=share/mysql -DINSTALL_MYSQLSHAREDIR=share/mysql -DINSTALL_MYSQLTESTDIR= -DINSTALL_DOCDIR=share/mysql/docs -DINSTALL_SHAREDIR=share/mysql 
-- Running cmake version 3.19.7
-- Could NOT find Git (missing: GIT_EXECUTABLE) 
-- CMAKE_MODULE_PATH is /tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/cmake
-- MySQL 8.0.26
-- The C compiler identification is Clang 7.1.0
-- The CXX compiler identification is Clang 7.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/jx31ah14p6pf488i7wwd88y2izmxi0wj-clang-wrapper-7.1.0/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/jx31ah14p6pf488i7wwd88y2izmxi0wj-clang-wrapper-7.1.0/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Source directory /private/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26
-- Binary directory /private/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build
-- CMAKE_GENERATOR: Unix Makefiles
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void *
-- Check size of void * - done
-- SIZEOF_VOIDP 8
CMake Error at cmake/package_name.cmake:92 (LIST):
  LIST GET given empty list
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:93 (LIST):
  LIST GET given empty list
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:95 (STRING):
  STRING sub-command REGEX, mode MATCH needs at least 5 arguments total to
  command.
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:98 (MESSAGE):
  Could not run sw_vers
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


-- Configuring incomplete, errors occurred!
See also "/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build/CMakeFiles/CMakeOutput.log".
```

</details>

### Steps To Reproduce
Steps to reproduce the behavior:
1. `nix build nixpkgs#mysql80`

### Expected behavior

`nix build nixpkgs#mysql80` succeeds.

### Additional context

### Notify maintainers
<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@orivej

### Metadata
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-darwin"`
 - host os: `Darwin 20.6.0, macOS 10.16`
 - multi-user?: `yes`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.4pre20210707_02dd6bb`
 - channels(cohei): `"nixpkgs-21.11pre304920.dd98b100651"`
 - channels(root): `"nixpkgs-21.11pre304920.dd98b100651"`
 - nixpkgs: `/Users/cohei/.nix-defexpr/channels/nixpkgs`
```

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
```

I hit the same issue since a week ago starting with mysql8.0.25 and today with mysql8.0.26

```
building '/nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv'...
unpacking sources
unpacking source archive /nix/store/8qlc0wxzndifrnqrckfjq5bznif1fx73-mysql-8.0.26.tar.gz
source root is mysql-8.0.26
setting SOURCE_DATE_EPOCH to timestamp 1625127339 of file mysql-8.0.26/sql/sql_yacc.h
patching sources
applying patch /nix/store/sn3mhjgqrrplgag8grhc91g5a9lcp1z2-abi-check.patch
patching file cmake/do_abi_check.cmake
patching file include/nostdinc/stdint.h
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/doc/mysql -DCMAKE_INSTALL_INFODIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/include -DCMAKE_INSTALL_SBINDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/strip -DCMAKE_RANLIB=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/ranlib -DCMAKE_AR=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26 -DCMAKE_SKIP_BUILD_RPATH=OFF -DFORCE_UNSUPPORTED_COMPILER=1 -DWITH_ROUTER=OFF -DWITH_SYSTEM_LIBS=ON -DWITH_UNIT_TESTS=OFF -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock -DMYSQL_DATADIR=/var/lib/mysql -DINSTALL_INFODIR=share/mysql/docs -DINSTALL_MANDIR=share/man -DINSTALL_PLUGINDIR=lib/mysql/plugin -DINSTALL_INCLUDEDIR=include/mysql -DINSTALL_DOCREADMEDIR=share/mysql -DINSTALL_SUPPORTFILESDIR=share/mysql -DINSTALL_MYSQLSHAREDIR=share/mysql -DINSTALL_MYSQLTESTDIR= -DINSTALL_DOCDIR=share/mysql/docs -DINSTALL_SHAREDIR=share/mysql
-- Running cmake version 3.19.7
-- Could NOT find Git (missing: GIT_EXECUTABLE)
-- CMAKE_MODULE_PATH is /tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/cmake
-- MySQL 8.0.26
-- The C compiler identification is Clang 7.1.0
-- The CXX compiler identification is Clang 7.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/jx31ah14p6pf488i7wwd88y2izmxi0wj-clang-wrapper-7.1.0/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/jx31ah14p6pf488i7wwd88y2izmxi0wj-clang-wrapper-7.1.0/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Source directory /private/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26
-- Binary directory /private/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build
-- CMAKE_GENERATOR: Unix Makefiles
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void *
-- Check size of void * - done
-- SIZEOF_VOIDP 8
CMake Error at cmake/package_name.cmake:92 (LIST):
  LIST GET given empty list
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:93 (LIST):
  LIST GET given empty list
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:95 (STRING):
  STRING sub-command REGEX, mode MATCH needs at least 5 arguments total to
  command.
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:98 (MESSAGE):
  Could not run sw_vers
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


-- Configuring incomplete, errors occurred!
See also "/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv' failed with exit code 1
```

**Metadata**
Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the result.
```
❯ nix-shell -p nix-info --run "nix-info -m"
these paths will be fetched (0.01 MiB download, 0.03 MiB unpacked):
  /nix/store/p5ggaaw5p95i54dkay1xwhddiylq1mvq-DarwinTools-1
  /nix/store/qz7l5hsnz1sb901fqcs9c79ranwnp9zs-nix-info
copying path '/nix/store/p5ggaaw5p95i54dkay1xwhddiylq1mvq-DarwinTools-1' from 'https://cache.nixos.org'...
copying path '/nix/store/qz7l5hsnz1sb901fqcs9c79ranwnp9zs-nix-info' from 'https://cache.nixos.org'...
 - system: `"x86_64-darwin"`
 - host os: `Darwin 20.6.0, macOS 10.16`
 - multi-user?: `no`
 - sandbox: `no`
 - version: `nix-env (Nix) 2.3.15`
 - channels(yeongsheng): `"darwin, nixpkgs-21.11pre305747.ab5b6828af2"`
 - nixpkgs: `/Users/yeongsheng/.nix-defexpr/channels/nixpkgs`
```

```
building '/nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv'...
unpacking sources
unpacking source archive /nix/store/8qlc0wxzndifrnqrckfjq5bznif1fx73-mysql-8.0.26.tar.gz
source root is mysql-8.0.26
setting SOURCE_DATE_EPOCH to timestamp 1625127339 of file mysql-8.0.26/sql/sql_yacc.h
patching sources
applying patch /nix/store/sn3mhjgqrrplgag8grhc91g5a9lcp1z2-abi-check.patch
patching file cmake/do_abi_check.cmake
patching file include/nostdinc/stdint.h
configuring
fixing cmake files...
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DCMAKE_SKIP_BUILD_RPATH=ON -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/doc/mysql -DCMAKE_INSTALL_INFODIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/share/man -DCMAKE_INSTALL_OLDINCLUDEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/include -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/include -DCMAKE_INSTALL_SBINDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_OSX_SYSROOT= -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/strip -DCMAKE_RANLIB=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/ranlib -DCMAKE_AR=/nix/store/falx4gakl6k1q64aqr4f8mvy8vfqcqaq-cctools-binutils-darwin-949.0.1/bin/ar -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_INSTALL_PREFIX=/nix/store/4a7gl3wwm8xl3g5msmll0wjpbdqq94lj-mysql-8.0.26 -DCMAKE_SKIP_BUILD_RPATH=OFF -DFORCE_UNSUPPORTED_COMPILER=1 -DWITH_ROUTER=OFF -DWITH_SYSTEM_LIBS=ON -DWITH_UNIT_TESTS=OFF -DMYSQL_UNIX_ADDR=/run/mysqld/mysqld.sock -DMYSQL_DATADIR=/var/lib/mysql -DINSTALL_INFODIR=share/mysql/docs -DINSTALL_MANDIR=share/man -DINSTALL_PLUGINDIR=lib/mysql/plugin -DINSTALL_INCLUDEDIR=include/mysql -DINSTALL_DOCREADMEDIR=share/mysql -DINSTALL_SUPPORTFILESDIR=share/mysql -DINSTALL_MYSQLSHAREDIR=share/mysql -DINSTALL_MYSQLTESTDIR= -DINSTALL_DOCDIR=share/mysql/docs -DINSTALL_SHAREDIR=share/mysql
-- Running cmake version 3.19.7
-- Could NOT find Git (missing: GIT_EXECUTABLE)
-- CMAKE_MODULE_PATH is /tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/cmake
-- MySQL 8.0.26
-- The C compiler identification is Clang 7.1.0
-- The CXX compiler identification is Clang 7.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/jx31ah14p6pf488i7wwd88y2izmxi0wj-clang-wrapper-7.1.0/bin/clang - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/jx31ah14p6pf488i7wwd88y2izmxi0wj-clang-wrapper-7.1.0/bin/clang++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Source directory /private/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26
-- Binary directory /private/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build
-- CMAKE_GENERATOR: Unix Makefiles
-- Looking for sys/types.h
-- Looking for sys/types.h - found
-- Looking for stdint.h
-- Looking for stdint.h - found
-- Looking for stddef.h
-- Looking for stddef.h - found
-- Check size of void *
-- Check size of void * - done
-- SIZEOF_VOIDP 8
CMake Error at cmake/package_name.cmake:92 (LIST):
  LIST GET given empty list
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:93 (LIST):
  LIST GET given empty list
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:95 (STRING):
  STRING sub-command REGEX, mode MATCH needs at least 5 arguments total to
  command.
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


CMake Error at cmake/package_name.cmake:98 (MESSAGE):
  Could not run sw_vers
Call Stack (most recent call first):
  cmake/package_name.cmake:156 (GET_PACKAGE_FILE_NAME)
  CMakeLists.txt:653 (INCLUDE)


-- Configuring incomplete, errors occurred!
See also "/tmp/nix-build-mysql-8.0.26.drv-0/mysql-8.0.26/build/CMakeFiles/CMakeOutput.log".
builder for '/nix/store/lg10b14df5qbmkfdd0padmb9rx0dc3m2-mysql-8.0.26.drv' failed with exit code 1
```

--- a/pkgs/servers/sql/mysql/8.0.x.nix
+++ b/pkgs/servers/sql/mysql/8.0.x.nix
@@ -1,6 +1,6 @@
 { lib, stdenv, fetchurl, bison, cmake, pkg-config
 , boost, icu, libedit, libevent, lz4, ncurses, openssl, protobuf, re2, readline, zlib, zstd
-, numactl, perl, cctools, CoreServices, developer_cmds, libtirpc, rpcsvc-proto, curl
+, numactl, perl, cctools, CoreServices, developer_cmds, libtirpc, rpcsvc-proto, curl, DarwinTools
 }:
 
 let
@@ -32,7 +32,7 @@ self = stdenv.mkDerivation rec {
   ] ++ lib.optionals stdenv.isLinux [
     numactl libtirpc
   ] ++ lib.optionals stdenv.isDarwin [
-    cctools CoreServices developer_cmds
+    cctools CoreServices developer_cmds DarwinTools
   ];
 
   outputs = [ "out" "static" ];

--- a/pkgs/top-level/all-packages.nix
+++ b/pkgs/top-level/all-packages.nix
@@ -20215,7 +20215,7 @@ with pkgs;
   };
 
   mysql80 = callPackage ../servers/sql/mysql/8.0.x.nix {
-    inherit (darwin) cctools developer_cmds;
+    inherit (darwin) cctools developer_cmds DarwinTools;
     inherit (darwin.apple_sdk.frameworks) CoreServices;
     boost = boost173; # Configure checks for specific version.
     protobuf = protobuf3_11;

